website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,373 @@
+## How to use this component
+
+!!! Insight
+
+While it’s not necessary to be familiar with CSS3 grid specifications to use this component, some knowledge may be helpful in achieving more complex layouts.
+
+A helpful reference with clear examples: [CSS Tricks: Complete grid layout guide](https://css-tricks.com/snippets/css/complete-guide-grid/).
+
+!!!
+
+The `Layout::Grid` and optional `Layout::Grid::Item` components provide a way to quickly build out flexible grid-based layouts of components or elements without needing to write a lot of custom CSS code or understand all the intricacies of CSS grid styles.
+
+### Basic usage
+
+The simplest way to implement a grid layout is by using the `Layout::Grid` component to wrap content directly. A grid layout of equal width “columns” is created by default.
+
+```handlebars
+<Hds::Layout::Grid class="doc-grid-outline-children">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+```
+
+Every child of the **grid container** will be stretched to fit evenly within the underlying grid column tracks behaving as a **grid item** (for details on what this means, refer to the guide linked at the top of the page).
+
+In some cases, it may be necessary to wrap one or more content items within the optional `Layout::Grid::Item` component. i.e., to group content together within a column or row, prevent content from being stretched to fit the underlying grid column width, or to make use of `rowspan` and `colspan` options in order to create more complex layouts. (See below for more details and examples on these features.)
+
+### Preventing content stretch
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="100%" @gap="16" as |LG|>
+  <Hds::Badge @text="Stretched badge" @color="critical" />
+
+  <LG.Item>
+    <Hds::Badge @text="Non-stretched badge" @color="success" />
+  </LG.Item>
+</Hds::Layout::Grid>
+```
+
+!!! Info
```

Comment:
Ahh I see you've moved it here. Mmm, I think it's a bit lost here, no?

Reply:
I'll move it.